Add kerberos troubleshooting (#11856)
DataDog/integrations-core: kafka_consumer/README.md

* TLS certificate to authenticate a secure connection to the Kafka broker
  * If JKS keystore is used, a certificate needs to be exported from the keystore and the file path should be configured with the `tls_cert` or `tls_ca_cert` options, as applicable. 
  * If a private key is required to authenticate the certificate, it should be configured with the `tls_private_key` option. If applicable, the private key password should be configured with the `tls_private_key_password`. 
* `KRB5_CLIENT_KTNAME` environment variable pointing to the Kafka client's Kerberos keytab location if it differs from the default path (i.e. `KRB5_CLIENT_KTNAME=/etc/krb5.keytab`)
* `KRB5CCNAME` environment variable pointing to the Kafka client's Kerberos credentials ticket cache if it differs from the default path (i.e. `KRB5CCNAME=/tmp/krb5cc_xxx`)
* If the Datadog Agent is unable to access the environment variables, configure the environment variables in a Datadog Agent service configuration override file:

**Linux**
1. Configure the environment variables in an environment file.
   For example: `/path/to/environment/file`
  ```
  KRB5_CLIENT_KTNAME=/etc/krb5.keytab
  KRB5CCNAME=/tmp/krb5cc_xxx

>   ```

Reviewer: blocks need one more space

- [Troubleshooting and Deep Dive for Kafka][10]
- [Agent failed to retrieve RMIServer stub][11]

**Kerberos GSSAPI Authentication**

Depending on your Kafka cluster's Kerberos setup, you may need to configure the following:

* Kafka client configured for the Datadog Agent to connect to the Kafka broker. The Kafka client should be added as a Kerberos principal and added to a Kerberos keytab. The Kafka client should also have a valid kerberos ticket. 
* TLS certificate to authenticate a secure connection to the Kafka broker.
  * If JKS keystore is used, a certificate needs to be exported from the keystore and the file path should be configured with the applicable `tls_cert` and/or `tls_ca_cert` options. 
  * If a private key is required to authenticate the certificate, it should be configured with the `tls_private_key` option. If applicable, the private key password should be configured with the `tls_private_key_password`. 
* `KRB5_CLIENT_KTNAME` environment variable pointing to the Kafka client's Kerberos keytab location if it differs from the default path (i.e. `KRB5_CLIENT_KTNAME=/etc/krb5.keytab`)

> * `KRB5CCNAME` environment variable pointing to the Kafka client's Kerberos credentials ticket cache if it differs from the default path (i.e. `KRB5CCNAME=/tmp/krb5cc_xxx`)

Reviewer 2: ```suggestion
* `KRB5CCNAME` environment variable pointing to the Kafka client's Kerberos credentials ticket cache if it differs from the default path (for example, `KRB5CCNAME=/tmp/krb5cc_xxx`)
```

* If the Datadog Agent is unable to access the environment variables, configure the environment variables in a Datadog Agent service configuration override file for your operating system. For example, in a Linux `systemd` environment: 

**Linux Systemd**

1. Configure the environment variables in an environment file.
   For example: `/path/to/environment/file`

  ```
  KRB5_CLIENT_KTNAME=/etc/krb5.keytab
  KRB5CCNAME=/tmp/krb5cc_xxx
  ```

> 2. Create a Datadog Agent service configuration override file: `sudo systemctl edit datadog-agent.service`

Reviewer 3: Are the agent commands different depending on OS?

Author: Since we are suggesting modifying the agent service unit files, the commands would indeed be different for each OS. These instructions are intended to be an example for systemd linux systems. Should I make it more explicit that this is just an example and that the steps may vary for other OSes?

3. Configure the following in the override file:

  ```
  [Service]
  EnvironmentFile=/path/to/environment/file
  ```

> 4. Run the following commands to reload the systemd daemon, datadog-agent service, and datadog agent:

Reviewer 2: ```suggestion
4. Run the following commands to reload the systemd daemon, datadog-agent service, and Datadog Agent:
```
